documentation/learning-to-rank.html

```diff
@@ -0,0 +1,275 @@
+---
+# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Please update copyright to "Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root."